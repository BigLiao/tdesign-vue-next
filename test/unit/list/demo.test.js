import { mount } from '@vue/test-utils';
import headerFooter from '@/examples/list/demos/header-footer.vue';
import imageText from '@/examples/list/demos/image-text.vue';
import multiline from '@/examples/list/demos/multiline.vue';
import operation from '@/examples/list/demos/operation.vue';
import scroll from '@/examples/list/demos/scroll.vue';
import size from '@/examples/list/demos/size.vue';

describe('Button', () => {
  it('headerFooter demo works fine', () => {
    const wrapper = mount(headerFooter);
    expect(wrapper.element).toMatchSnapshot();
  });
  it('imageText demo works fine', () => {
    const wrapper = mount(imageText);
    expect(wrapper.element).toMatchSnapshot();
  });
  it('multiline demo works fine', () => {
    const wrapper = mount(multiline);
    expect(wrapper.element).toMatchSnapshot();
  });
  it('operation demo works fine', () => {
    const wrapper = mount(operation);
    expect(wrapper.element).toMatchSnapshot();
  });
  it('scroll demo works fine', () => {
    const wrapper = mount(scroll);
    expect(wrapper.element).toMatchSnapshot();
  });
  it('size demo works fine', () => {
    const wrapper = mount(size);
    expect(wrapper.element).toMatchSnapshot();
  });
});
