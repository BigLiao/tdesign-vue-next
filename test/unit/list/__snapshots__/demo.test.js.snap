// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button headerFooter demo works fine 1`] = `
<div>
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <div
      class="t-list__header"
    >
      这里是 Header
    </div>
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      
    </ul>
    <div
      class="t-list__footer"
    >
      这里是 Footer
    </div>
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
  <div
    style="margin-bottom: 16px;"
  />
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <div
      class="t-list__header"
    >
      
       通过 Slot 插入的 Header 
      
    </div>
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      
    </ul>
    <div
      class="t-list__footer"
    >
      
       通过 Slot 插入的 Footer 
      
    </div>
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
</div>
`;

exports[`Button imageText demo works fine 1`] = `
<div>
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
  <div
    style="margin-bottom: 16px;"
  />
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                
                <p>
                  列表内容列表内容列表内容
                </p>
                <p>
                  列表内容列表内容列表内容
                </p>
                <p>
                  列表内容列表内容列表内容
                </p>
                <p>
                  列表内容列表内容列表内容
                </p>
                <p>
                  列表内容列表内容列表内容
                </p>
                
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
</div>
`;

exports[`Button multiline demo works fine 1`] = `
<div>
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <!---->
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <!---->
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <!---->
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <!---->
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
</div>
`;

exports[`Button operation demo works fine 1`] = `
<div>
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <li
            class="t-list-item__action"
          >
            
            <li>
              <a
                href=""
              >
                操作1
              </a>
            </li>
            <li>
              <a
                href=""
              >
                操作2
              </a>
            </li>
            <li>
              <a
                href=""
              >
                操作3
              </a>
            </li>
            
          </li>
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <li
            class="t-list-item__action"
          >
            
            <li>
              <a
                href=""
              >
                操作1
              </a>
            </li>
            <li>
              <a
                href=""
              >
                操作2
              </a>
            </li>
            <li>
              <a
                href=""
              >
                操作3
              </a>
            </li>
            
          </li>
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
  <div
    style="margin-bottom: 16px;"
  />
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <li
            class="t-list-item__action"
          >
            
            <li>
              <svg
                class="t-icon t-icon-check-circle-filled"
                version="1.1"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 96C741.7504576 96 928 282.24954432 928 512 928 741.7504576 741.7504576 928 512 928 282.24954432 928 96 741.7504576 96 512 96 282.24954432 282.24954432 96 512 96ZM690.8655168 353.58818368L464.56401024 579.82818368 328.82684416 444.09785152 283.57201024 489.35268544 464.59134592 670.3720192 509.84617984 625.1171872 736.120352 398.8430176 690.8655168 353.58818368Z"
                  fill-opacity=".9"
                  fill-rule="evenodd"
                />
              </svg>
            </li>
            <li>
              <svg
                class="t-icon t-icon-check-circle-filled"
                version="1.1"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 96C741.7504576 96 928 282.24954432 928 512 928 741.7504576 741.7504576 928 512 928 282.24954432 928 96 741.7504576 96 512 96 282.24954432 282.24954432 96 512 96ZM690.8655168 353.58818368L464.56401024 579.82818368 328.82684416 444.09785152 283.57201024 489.35268544 464.59134592 670.3720192 509.84617984 625.1171872 736.120352 398.8430176 690.8655168 353.58818368Z"
                  fill-opacity=".9"
                  fill-rule="evenodd"
                />
              </svg>
            </li>
            <li>
              <svg
                class="t-icon t-icon-check-circle-filled"
                version="1.1"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 96C741.7504576 96 928 282.24954432 928 512 928 741.7504576 741.7504576 928 512 928 282.24954432 928 96 741.7504576 96 512 96 282.24954432 282.24954432 96 512 96ZM690.8655168 353.58818368L464.56401024 579.82818368 328.82684416 444.09785152 283.57201024 489.35268544 464.59134592 670.3720192 509.84617984 625.1171872 736.120352 398.8430176 690.8655168 353.58818368Z"
                  fill-opacity=".9"
                  fill-rule="evenodd"
                />
              </svg>
            </li>
            
          </li>
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          <div
            class="t-list-item__meta"
          >
            
            <div
              class="t-list-item__meta-avatar"
            >
              <img
                src="https://tdesign.gtimg.com/list-icon.png"
              />
            </div>
            <div
              class="t-list-item__meta-content"
            >
              <h3
                class="t-list-item__meta-title"
              >
                列表主内容
              </h3>
              <p
                class="t-list-item__meta-description"
              >
                列表内容列表内容
              </p>
            </div>
            
          </div>
          
          <li
            class="t-list-item__action"
          >
            
            <li>
              <svg
                class="t-icon t-icon-check-circle-filled"
                version="1.1"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 96C741.7504576 96 928 282.24954432 928 512 928 741.7504576 741.7504576 928 512 928 282.24954432 928 96 741.7504576 96 512 96 282.24954432 282.24954432 96 512 96ZM690.8655168 353.58818368L464.56401024 579.82818368 328.82684416 444.09785152 283.57201024 489.35268544 464.59134592 670.3720192 509.84617984 625.1171872 736.120352 398.8430176 690.8655168 353.58818368Z"
                  fill-opacity=".9"
                  fill-rule="evenodd"
                />
              </svg>
            </li>
            <li>
              <svg
                class="t-icon t-icon-check-circle-filled"
                version="1.1"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 96C741.7504576 96 928 282.24954432 928 512 928 741.7504576 741.7504576 928 512 928 282.24954432 928 96 741.7504576 96 512 96 282.24954432 282.24954432 96 512 96ZM690.8655168 353.58818368L464.56401024 579.82818368 328.82684416 444.09785152 283.57201024 489.35268544 464.59134592 670.3720192 509.84617984 625.1171872 736.120352 398.8430176 690.8655168 353.58818368Z"
                  fill-opacity=".9"
                  fill-rule="evenodd"
                />
              </svg>
            </li>
            <li>
              <svg
                class="t-icon t-icon-check-circle-filled"
                version="1.1"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 96C741.7504576 96 928 282.24954432 928 512 928 741.7504576 741.7504576 928 512 928 282.24954432 928 96 741.7504576 96 512 96 282.24954432 282.24954432 96 512 96ZM690.8655168 353.58818368L464.56401024 579.82818368 328.82684416 444.09785152 283.57201024 489.35268544 464.59134592 670.3720192 509.84617984 625.1171872 736.120352 398.8430176 690.8655168 353.58818368Z"
                  fill-opacity=".9"
                  fill-rule="evenodd"
                />
              </svg>
            </li>
            
          </li>
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
</div>
`;

exports[`Button scroll demo works fine 1`] = `
<div
  class="t-list t-size-m"
  style="max-height: 200px;"
>
  
  
  <!---->
  <ul
    class="t-list-items"
  >
    
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    <li
      class="t-list-item"
    >
      <div
        class="t-list-item-main"
      >
        
        列表内容列表内容
        
        <!---->
      </div>
    </li>
    
  </ul>
  <!---->
  
  <div
    class="t-list__load"
  >
    <!---->
  </div>
  
</div>
`;

exports[`Button size demo works fine 1`] = `
<div>
  <div
    class="t-list t-size-s t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
  <div
    style="margin-bottom: 16px;"
  />
  <div
    class="t-list t-size-m t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
  <div
    style="margin-bottom: 16px;"
  />
  <div
    class="t-list t-size-l t-list--split"
  >
    
    
    <!---->
    <ul
      class="t-list-items"
    >
      
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      <li
        class="t-list-item"
      >
        <div
          class="t-list-item-main"
        >
          
          列表内容列表内容
          
          <!---->
        </div>
      </li>
      
    </ul>
    <!---->
    
    <div
      class="t-list__load"
    >
      <!---->
    </div>
    
  </div>
</div>
`;
